pyro/distributions/extended.py

```diff
@@ -23,7 +23,7 @@ class ExtendedBinomial(Binomial):
 
     def log_prob(self, value):
         result = super().log_prob(value)
-        invalid = ~super().support.check(value)
+        invalid = (value < 0) | (value > self.total_count)
```

Comment:
curous: why this change?

Reply:
This is a little cheaper. The `integer_interval` additionally checks `value % 1 == 0`, but that is already checked in validation by the above line, and doesn't affect numerical stability.